src/Events/Integrations/Plugins/Tickets_Wallet_Plus/Passes/Apple_Wallet/Event_Data_Add_Info.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\Tickets_Wallet_Plus\Passes\Apple_Wallet;
+
+/**
+ * Class Event_Data_Add_Info
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Tickets_Wallet_Plus\Passes\Apple_Wallet
+ */
+class Event_Data_Add_Info {
+
+	/**
+	 * Add the Event date into the Apple Pass `back` data.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $pass_data The existing pass data.
+	 * @param array $attendee  The attendee information.
+	 *
+	 * @return array Modified pass data.
+	 */
+	public function add_event_date_to_apple_pass_data( array $pass_data, array $attendee ): array {
```

Comment:
We dont need to mention Apple pass again, we are in the integration folder for Apple Wallet.
maybe `include_data()` since we are also in the Event Modifier class.